src/_layouts/service_line.liquid

```diff
@@ -3,6 +3,8 @@ layout: default
 ---
 
 {% assign translations = site.translations[site.lang].[page.service-line] %}
+{%- assign all-service-lines = site.translations[site.lang].all-service-lines -%}
```

Comment:
It's not clear to me why we need to assign `all-service-lines` here (or at all really)

Reply:
I have assigned it both here and within the index page for Software Modernisation, as the index page doesn't pick it up from the Service Line Template. 

I was thinking that when we move the markup from the index page into the template, then we can remove the assignment within the index page. 

I can remove this assignment here and we can just swap out where the assignment is made when we come to moving the markup from the softmod index page into the Service Line Template.

Reply:
Please see my other comment referencing assignments not being picked up by anything being pulled in through `{{ content }}`